test/SlowTests/Issues/RavenDB_20084.cs

```diff
@@ -47,119 +41,154 @@ public async Task ShouldUpdateBackupInfoIfDatabaseUnloaded()
             customSettings: new Dictionary<string, string>()
             {
                 [RavenConfiguration.GetKey(x => x.Databases.MaxIdleTime)] = "1",
-                [RavenConfiguration.GetKey(x => x.Databases.FrequencyToCheckForIdle)] = "1"
+                [RavenConfiguration.GetKey(x => x.Databases.FrequencyToCheckForIdle)] = "1",
+                [RavenConfiguration.GetKey(x => x.Logs.Mode)] = nameof(LogMode.Information)
```

Comment:
this is a  global change, you cannot do that

Reply:
Maybe we shouldn't reuse the servers?

Reply:
Logger configuration is static.

Reply:
Okay, perhaps the `Operations` logging level will suffice for this scenario...
Is the logging level set to `Operations` in the test environment?

Reply:
subscribing a stream will switch the logging level, but do not rely on this, you will turn on all the logs for all the tests, add some ForTestingPurposes code IMO

Reply:
Done.